### ISSUE TITLE
Add MCC metrics to evaluate data in addition to accuracy.

### ISSUE THREAD

**Issue by GabrieleMeoni** [GabrieleMeoni/END2END#9]
[MCC](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html) metric should be better to take into account of unbalanced classes.
